library/nis.po

```diff
@@ -6,33 +6,38 @@ msgstr ""
 "Project-Id-Version: Python 3.6\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2017-04-02 22:11+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2018-07-12 21:39+0200\n"
 "Language-Team: FRENCH <traductions@lists.afpy.org>\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: \n"
+"X-Generator: Poedit 1.8.11\n"
 
 #: ../Doc/library/nis.rst:3
 msgid ":mod:`nis` --- Interface to Sun's NIS (Yellow Pages)"
-msgstr ""
+msgstr "mod:`nis` --- Interface dans Sun's NIS (pages jaunes)"
 
 #: ../Doc/library/nis.rst:14
 msgid ""
 "The :mod:`nis` module gives a thin wrapper around the NIS library, useful "
 "for central administration of several hosts."
 msgstr ""
+"The :mod:`nis` module fournit une enveloppe fine (thin wrapper) autour de la "
+"librairie NIS, utile pour l'administration centralisée de plusieurs hôtes."
 
 #: ../Doc/library/nis.rst:17
 msgid ""
 "Because NIS exists only on Unix systems, this module is only available for "
 "Unix."
 msgstr ""
+"Du fait que NIS existe seulement sur les systèmes Unix, ce module est "
+"seulement disponible pour Unix."
 
 #: ../Doc/library/nis.rst:19
 msgid "The :mod:`nis` module defines the following functions:"
-msgstr ""
+msgstr "The :mod:`nis` module définit les instructions suivantes:"
```

Comment:
s/The :mod:`nis` module/Le module :mod:`nis`/

Reply:
:+1: